src/network/netdev/tunnel.c

```diff
@@ -665,6 +739,38 @@ int config_parse_6rd_prefix(const char* unit,
         return 0;
 }
 
+int config_parse_erpsan_index(const char* unit,
+                              const char *filename,
+                              unsigned line,
+                              const char *section,
+                              unsigned section_line,
+                              const char *lvalue,
+                              int ltype,
+                              const char *rvalue,
+                              void *data,
+                              void *userdata) {
+        Tunnel *t = userdata;
+        uint32_t k = 0;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = safe_atou32(rvalue, &k);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse erspan index option, ignoring: %s", rvalue);
+                return 0;
+        }
+
+        if (k >= (1 << 20) || k == 0)
```

Comment:
I guess maybe you missed my previous comment. I think the check should be moved to `tunnel_verify()`. At least, we need to check erspan_index is valid in `tunnel_verify()`. Or, do I misunderstand something?

Reply:
right . moved now.